packages/snap-preact-components/src/components/Molecules/RadioList/RadioList.tsx

```diff
@@ -116,30 +106,16 @@ export function RadioList(properties: RadioListProps): JSX.Element {
 			<div {...styling} className={classnames('ss__radio-list', className)}>
 				{titleText && <h5 className="ss__radio-list__title">{titleText}</h5>}
 
-				<ul className="ss__radio-list__options-wrapper">
+				<ul className={`ss__radio-list__options-wrapper ${disabled ? 'ss__radio-list__options-wrapper--disabled' : ''}`}>
```

Comment:
Similar to the "List" component, I think it is better to put the `--disabled` modifier class on the root element.